scripts/docs/sources2html.pl

```diff
@@ -1318,17 +1322,20 @@ sub get_vcf_content_types {
   my $genotypes = `tabix $file_full_path -H | grep '##FORMAT' | grep 'ID=GT'`;
   push @types, "genotype" if $genotypes;
   
+  my $chr = `tabix $file_full_path -l | head -n 1`;
+  chop $chr;
```

Comment:
Shouldn't we use `chomp` instead of `chop`? `chomp` only removes a newline character whereas `chop` removes the last character, no matter which one it is.

```suggestion
  chomp $chr;
```